### DISCUSSION TITLE
Who has latest CURL library debug x64

### DISCUSSION THREAD

**Discussion by GamebP** [curl/curl#13594] [General]
I'we checked the simple way to make the curl library. It had about 18steps. 
Is there any pre-build debug x64 curl library out there that someone could give?

**Comment by dfandrich** [comment]
I count three steps:

1. ./configure
2. make
3. make install

All the binaries we know about are listed at https://curl.se/download.html

**Comment by GamebP** [comment]
ok, ill try this way

**Comment by GamebP** [comment]
Dosen't work

**Comment by GamebP** [comment]
I desperately need it.

**Comment by danielgustafsson** [comment]
It's still only three steps, but you need to create the `configure` binary first:
1. autoreconf -fi
2. ./configure
3. make install

**Comment by GamebP** [comment]
is this even windows 10?

**Reply by bagder** [reply]
This is the first mention of windows 10. I believe our mind-reader system glitched. Do you still want us to keep guessing what you want?

**Comment by GamebP** [comment]
I just need curllib for debug x64 for my project that gets data from github text. Why is it so hard to make it??

**Comment by jay** [comment]
If you want a debug build of libcurl in Windows you'll have to build it yourself. There's several ways to build libcurl in Windows and how you do it depends on what build tools you have. The easiest is probably [winbuild](https://github.com/curl/curl/tree/curl-8_7_1/winbuild) if you have Visual Studio. Open a Visual Studio Command Prompt x64 (or x86 platform depending on what your app uses) which should be a Windows shortcut in the start/app menu, switch to the winbuild directory, and run `nmake /f Makefile.vc mode=dll DEBUG=yes`

**Comment by GamebP** [comment]
Alright I will try that later.

Jay Satiro ***@***.***> (šajā datumā: svētd., 2024. g. 12.
maijs 22:30) rakstīja:
